source/_integrations/aidot.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: Aidot
+description: Instructions on integrating Aidot home security with Home Assistant.
+ha_category:
+  - Light
+ha_release: 1.0.0
+ha_iot_class: local_polling
+ha_config_flow: true
+ha_codeowners:
+  - '@sz411'
+ha_domain: Aidot
+ha_homekit: true
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+The [Aidot](https://www.aidot.com/) integration allows you to control the Wi-Fi lights with home assistant.
+
+{% include integrations/config_flow.md %}
+
+## Supported Devices
```

Comment:
```suggestion
## Supported devices
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headers.